deployment/environment.go

```diff
@@ -50,6 +50,7 @@ type OffchainClient interface {
 type Chain struct {
 	// Selectors used as canonical chain identifier.
 	Selector uint64
+	Name     string
```

Comment:
I worry this could be populated wrong, can we leave it as is and derive from selector?

Reply:
I can move the name population to deployment.environment ow for every function we will have to call the chaininfo function,

Reply:
could we just have a method on Chain called Name() which does the derivation?